examples/navigation/containers/Admin.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export default function Admin() {
+  return (
+    <p>
+      You don't have access. Redirecting back home.
+    </p>
+  );
+}
```

Comment:
You'll want to rename these files to `.jsx`, if for no other reason than to get GitHub support for JSX formatting.